deploy/cloud/scripts/init.sh

```diff
@@ -139,7 +211,8 @@ function sealos_run_frontend {
     --config-file etc/sealos/desktop-config.yaml
 
   # sealos authorize !!must run after sealos_run_controller frontend-desktop.tar and before sealos_run_frontend
-  sealos_authorize
+  # TODO fix sealos_authorize in controller/job/init
```

Comment:
@bxy4543 fix sealos init admin user logic and add region data to CockroachDB in job/init